# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: image/test/gtest/TestDecoders.cpp
# Commit: da557d1806dd
# Full Hash: da557d1806dd9e1e519e00a4481b2e075f722ae8
# Author: Butkovits Atila <abutkovits@mozilla.com>
# Date: 2023-01-27 22:54 +0200
# Regressor Bug: 1788119
# File Overlap Count: 1
# Overlapping Files: Cargo.lock
# Description:
#   Backed out 5 changesets (bug 1788119) for causing bustages complaining about AVIFDecodedData. CLOSED TREE
# ==============================================================================

diff -r d980bebc6d26 -r da557d1806dd image/test/gtest/TestDecoders.cpp
--- a/image/test/gtest/TestDecoders.cpp	Fri Jan 27 20:35:34 2023 +0000
+++ b/image/test/gtest/TestDecoders.cpp	Fri Jan 27 22:54:08 2023 +0200
@@ -917,18 +917,6 @@
   CheckAnimationDecoderSingleChunk(GreenFirstFrameAnimatedWebPTestCase());
 }
 
-TEST_F(ImageDecoders, AnimatedAVIFSingleChunk) {
-  CheckDecoderSingleChunk(GreenFirstFrameAnimatedAVIFTestCase());
-}
-
-TEST_F(ImageDecoders, AnimatedAVIFMultiChunk) {
-  CheckDecoderMultiChunk(GreenFirstFrameAnimatedAVIFTestCase());
-}
-
-TEST_F(ImageDecoders, AnimatedAVIFWithBlendedFrames) {
-  CheckAnimationDecoderSingleChunk(GreenFirstFrameAnimatedAVIFTestCase());
-}
-
 TEST_F(ImageDecoders, CorruptSingleChunk) {
   CheckDecoderSingleChunk(CorruptTestCase());
 }